docs/lib/storage/fragments/js/upload.md

```diff
@@ -60,67 +52,58 @@ const SSECustomerAlgorithm = 'string';
 const SSECustomerKey = new Buffer('...') || 'string';
 const SSECustomerKeyMD5 = 'string';
 const SSEKMSKeyId = 'string';
-Storage.put('test.txt', 'File content', {
+
+const result = await Storage.put('test.txt', 'File content', {
     serverSideEncryption, SSECustomerAlgorithm, SSECustomerKey, SSECustomerKeyMD5, SSEKMSKeyId
-})
-.then (result => console.log(result))
-.catch (err => console.log(err));
+});
 ```
 
 Other options available are:
 
 ```javascript
 Storage.put('test.txt', 'My Content', {
     acl: 'public-read', // https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#upload-property
-    cacheControl: '', // (String) Specifies caching behavior along the request/reply chain
-    contentDisposition: '', // (String) Specifies presentational information for the object
+    cacheControl: 'no-cache', // (String) Specifies caching behavior along the request/reply chain
+    contentDisposition: 'attachment', // (String) Specifies presentational information for the object
     expires: new Date().now() + 60 * 60 * 24 * 7, // (Date) The date and time at which the object is no longer cacheable. ISO-8601 string, or a UNIX timestamp in seconds
     metadata: { key: 'value' }, // (map<String>) A map of metadata to store with the object in S3.
 })
-.then (result => console.log(result))
-.catch(err => console.log(err));
 ```
 
 ## Browser uploads
 Upload an image in the browser:
 
 ```javascript
-class S3ImageUpload extends React.Component {
-  onChange(e) {
-      const file = e.target.files[0];
-      Storage.put('example.png', file, {
-          contentType: 'image/png'
-      })
-      .then (result => console.log(result))
-      .catch(err => console.log(err));
-  }
-
-  render() {
-      return (
-          <input
-              type="file" accept='image/png'
-              onChange={(evt) => this.onChange(evt)}
-          />
-      )
-  }
+function onChange(e) {
+  const file = e.target.files[0];
+  try {
+    await Storage.put(file.name, file, {
+      contentType: 'image/png' // contentType is optional
+    });
+  } catch (err) {
+    console.log('Error uploading file: ', error);
+  }  
 }
+
+<input
+  type="file"
+  onChange={onChange}
```

Comment:
Dont we need to call here? 
```suggestion
  onChange={this.onChange}
```

Reply:
So I tried to refactor out of anything framework specific, so if they are using React hooks then they should not have to worry about the scope or `this`